tsconfig.json

```diff
@@ -43,7 +43,7 @@
     // "noFallthroughCasesInSwitch": true,    /* Report errors for fallthrough cases in switch statement. */
 
     /* Module Resolution Options */
-    "moduleResolution": "node16",            /* Specify module resolution strategy: 'node' (Node.js) or 'classic' (TypeScript pre-1.6). */
+    "moduleResolution": "nodenext",            /* Specify module resolution strategy: 'node' (Node.js) or 'classic' (TypeScript pre-1.6). */
```

Comment:
The comment looks obsolete, consider changing it. FWIW, the link to the docs is https://www.typescriptlang.org/tsconfig#moduleResolution